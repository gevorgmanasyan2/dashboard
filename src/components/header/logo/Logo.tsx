import React, { FC } from 'react'
import HeaderLogo from '../../../assets/icons/logo.png'
import HeaderMobileLogo from '../../../assets/icons/mobile-logo.png'
import './logo.scss'
import { Link } from 'react-router-dom'

const Logo: FC = () => {
  return (
    <div className="logo__container">
      <Link to="/">
        <img src={HeaderLogo} alt="logo" className="logo__container__image" />
        <img src={HeaderMobileLogo} alt="logo" className="logo__container__mobile" />
      </Link>
    </div>
  )
}

export default Logo
