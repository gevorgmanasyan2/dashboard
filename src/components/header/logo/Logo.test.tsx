/* eslint-disable no-undef */
import React from 'react'
import { shallow } from 'enzyme'
import Logo from './Logo'

it('should render Logo component', () => {
  expect(shallow(<Logo />)).toMatchSnapshot()
})
