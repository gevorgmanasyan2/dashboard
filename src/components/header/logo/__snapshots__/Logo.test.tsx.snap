// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Logo component 1`] = `
<div
  className="logo__container"
>
  <Link
    to="/"
  >
    <img
      alt="logo"
      className="logo__container__image"
      src="test-file-stub"
    />
    <img
      alt="logo"
      className="logo__container__mobile"
      src="test-file-stub"
    />
  </Link>
</div>
`;
